shiftcommerce/shift_commerce-secure_trading
# updated values to use new secure trading design

### The issue

Matalan would like to try out the new secure trading iframe https://github.com/shiftcommerce/matalan-comms/issues/445

### The fix

Update the values:

```
The value of stprofile should equal paymentcard
The value of stdefaultprofile should equal st_paymentcardonly
The initial post url should be /process/payments/details
```

### QA

Using a front end that uses this branch, with the cookie `mmPaymentForm` set to "1".
proceed through checkout,
the iframe should look like this:

![screen shot 2018-09-13 at 08 47 01](https://user-images.githubusercontent.com/15713366/45474410-a12a8900-b731-11e8-8b10-069eaff134aa.png)
